source/HowToBuild.md

```diff
@@ -0,0 +1,40 @@
+# How To Build
+
+The build system for this project is based on [Max-SDK-Lite](https://github.com/tap/max-sdk-lite).
+
+
+## Prerequisites
+
+To build the externals in this package you will need some form of compiler support on your system. 
+
+* On the Mac this means Xcode (you can get from the App Store for free). 
+* On Windows this most likely means some version of Visual Studio (the free versions should work fine).
+
+You will also need to install [CMake](https://cmake.org/download/).
+
+
+## Building
+
+0. Get the code from Github, or download a zip and unpack it into a folder.
+1. In the Terminal or Console app of your choice, change directories (cd) into that folder.
```

Comment:
Maybe that is just because I was coming at it from the old setup. Fresh eyes would be less likely confused.